compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -1018,7 +1018,7 @@ def cmd_start_endpoint(
 
             log.debug("Convey credentials; redirect stdout, stderr (to '%s')", ep_log)
             log_fd_flags = os.O_CREAT | os.O_WRONLY | os.O_APPEND | os.O_SYNC
-            log_fd = os.open(ep_log, log_fd_flags, mode=0o200)
+            log_fd = os.open(ep_log, log_fd_flags, mode=0o600)
```

Comment:
Odd ... this is already `0o600` in `main`.  Why is this showing as a change?

Reply:
Yes that is weird.  Must be a caching issue.  After I rebase again we'll see if it's still there.

Reply:
It's gone, so it was just an artifact.